source/variables/breakpoints.css

```diff
@@ -0,0 +1,13 @@
+/* 
+The breakpoints for all resolutions represented as pairs with the breakpoint name and its associated pixel width.
+http://cssnext.io/features/#custom-media-queries 
+@media (--medium) { ..styles.. }
+*/
+:root {
+	@custom-media --breakpoint--extra-small (width <= 320px) and (width >= 0); /* Used for style that should not cascade out */
```

Comment:
Are these being consumed as pixel values everywhere? Has best practice regarding pixels versus relative values in media queries changed? (Asking seriously. School me.)